src/main/java/kernbeisser/Windows/LogIn/LogInModel.java

```diff
@@ -27,11 +27,10 @@ public static User getLoggedIn() {
   }
 
   public static User getLoggedInFromDB() {
-    @Cleanup EntityManager em = DBConnection.getEntityManager();
-    @Cleanup(value = "commit")
-    EntityTransaction et = em.getTransaction();
-    et.begin();
-    return em.find(User.class, loggedIn.getId());
+    loggedIn = (User.getById(loggedIn.getId()));
+    Access.setDefaultManager(AccessManager.NO_ACCESS_CHECKING);
```

Comment:
Ist bestimmt nur ausversehen noch drinne oder weil wird ja in der nächsten Zeile überschrieben.